src/applications/static-pages/sass/static-pages.scss

```diff
@@ -77,3 +77,10 @@
     display: none;
   }
 }
+
+
+// Makes anything look like an h3
+.h3 {
+  font-size: 2rem;
+  font-weight: 700;
+}
```

Comment:
I think we actually have `.heading-level-3` for this 🙂 I think it's missing from our design system docs though.

@jaredcunha-usds Do you think it makes sense to include those headings classes under [utilities](https://department-of-veterans-affairs.github.io/vets-design-system-documentation/utilities/)?

Reply:
@ncksllvn - There is a `font-size` utility. I added some redundant classes to reflect that the font size refers to base styles of specific headings: https://department-of-veterans-affairs.github.io/vets-design-system-documentation/utilities/font-size

For these types of utilities, I tend to prefer the naming convention that describes exactly the CSS property it uses.